README.md

```diff
@@ -108,354 +108,354 @@ in order to add your own service into Lucida. Let's break it down into two steps
-    
-    Notice that we do not define which nodes the current node is pointed to by, so we do not know which node is pointed to
-    by the command center. Thus, we need to specify the starting nodes separately.
-    This is done by an optional second parameter in the constructor of `Graph`:
-    
-    ```
-    def __init__(self, node_list, starting_indices=[0]):
-    ```
-    
-    As you see, by default, the command center assumes the 0th node in the node list is the starting node.
-    Thus, a `QuerySpec` like the following will be sent to the `IMM` service:
-    
-    ```
-    { name: "query", 
-    content: [
-    { type: "image",
-    data: [ "1234567...abcdefg" ],
-    tags: ["localhost", "8082", "1", 1"] },
-    { type: "text",
-    data: [ "How old is this person?" ],
-    tags: ["localhost", "8083", "0"] } ] }
-    ```
-    
-    The `IMM` service receives this `QuerySpec` along with `LUCID`, and is responsible for further sending
-    the request to the `QA` service. The `IMM` service is allowed to modify the `QuerySpec` and
-    send a reconstructed `QuerySpec` to `QA`, but as long as `IMM` finally returns a `string` to the
-    command center, it is fine. This degree of flexibility opens up opportunities for complicated communication
-    between your services. Usually, a graph with one node suffices, because that one node may be an entry point
-    to a cluster of your services, which may have complicated feedback loops and use a different communication mechanism.
-    
-    Notice that there is only one possible prediction result if the user only gives image input: `class_IMM`,
-    because the query classifier only works on text input.
-    However, you can still send the image to multiple services like this:
-    
-    ```
-    ''image' : { 'class_IMM' : Graph([Node('IMM'), Node('IMC'), Node('IMAGE_CAPTIONING')], [0, 1, 2]) }, ...
-    ```
-    
-    Be aware that there are other parameters that you can change in this configuration file,
-    which are pretty self-explanatory in their names and comments.
-    
-  2. Add training data for your own query class.
-  
-    We already prepare some sample training data in [`lucida/commandcenter/data/`](lucida/commandcenter/data/),
-    but if you need to define a custom type of query that your service can handle,
-    you should create the following file in the above directory:
-  
-    ```
-    class_<NAME_OF_YOUR_QUERY_CLASS>.txt
-    ```
-    
-    , and have at least 40 pieces of text in it, each being one way to ask about the same question.
+      We already prepare some sample training data in [`lucida/commandcenter/data/`](lucida/commandcenter/data/),
+      but if you need to define a custom type of query that your service can handle,
+      you should create the following file in the above directory:
+
+      ```
+      class_<NAME_OF_YOUR_QUERY_CLASS>.txt
+      ```
+
+      , and have at least 40 pieces of text in it, each being one way to ask about the same question.
     
```

Comment:
This is definitely an improvement from before. Can you cut down the words in this README to make it more concise? Some information may not even be correct with the recent changes.